packages/stats/gbstats/gen_notebook.py

```diff
@@ -70,12 +71,18 @@ def create_notebook(
         ),
         nbf.new_markdown_cell("## Notebook Setup"),
         nbf.new_code_cell(
-            "# Requires gbstats version 0.4.0 or higher\n"
+            "# Requires gbstats version 0.5.0\n"
+            "from importlib.metadata import version\n"
+            "if version('gbstats') != '0.5.0':\n"
+            "    raise ValueError(\n"
+            '        f"""Current gbstats version: {version(\'gbstats\')}.\n'
+            '        Use `pip install gbstats` to upgrade to 0.5.0 from PyPI"""\n'
+            "    )\n"
```

Comment:
This seems a little too strict.  What if someone downloads a notebook, but then we release gbstats `0.5.1` and the person upgrades?  The old notebook they downloaded will stop working.

It almost seems like this logic should be reversed.  The generated notebook should have a version and it should ask the `gbstats` library if it's supported.

```python
if not gbstats.supports_notebook_version("0.5.0"):
  raise ValueError(...)
```